packages/types/src/Metadata/v5/Storage.ts

```diff
@@ -0,0 +1,262 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AnyNumber } from '../../types';
+
+import { assert } from '@polkadot/util';
+
+import Enum from '../../codec/Enum';
+import Struct from '../../codec/Struct';
+import Vector from '../../codec/Vector';
+import Bool from '../../primitive/Bool';
+import Bytes from '../../primitive/Bytes';
+import StorageHasher from '../../primitive/StorageHasher';
+import Text from '../../primitive/Text';
+import Type from '../../primitive/Type';
+import { PlainType, StorageFunctionModifier } from '../v3/Storage';
```

Comment:
Better import/export from v4, so that we know what changed in each version in one glance

Reply:
done.